packages/react-scripts/fixtures/kitchensink/src/features/syntax/ClassProperties.test.js

```diff
@@ -5,6 +5,8 @@ import ClassProperties from './ClassProperties';
 describe('class properties', () => {
   it('renders without crashing', () => {
     const div = document.createElement('div');
-    ReactDOM.render(<ClassProperties />, div);
+    return new Promise(resolve => {
+      ReactDOM.render(<ClassProperties notifyRendered={resolve} />, div);
```

Comment:
The notifyRendered function isn't called until after a re-render in the subcomponent -- so we don't want the initial render callback. Perhaps the naming change will make this more clear.